notebooks/official/custom/custom_training_container_and_model_registry.ipynb

```diff
@@ -401,18 +334,18 @@
         "\n",
         "Otherwise specify `(None, None)` to use a container image to run on a CPU.\n",
         "\n",
-        "Learn more [here](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators) hardware accelerator support for your region"
+        "Learn more [here](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators) hardware accelerator support for your location"
```

Comment:
Don't use "here" as link text. Recommend changing to:

- Learn more about [hardware accelerator support ](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators) for your location"

Reply:
Did required changes